domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/behandling/Førstegangsbehandling.kt

```diff
@@ -339,4 +349,63 @@ data class Førstegangsbehandling(
             this + neste
         }
     }
+
+    private fun lagFristForFramsettingAvKravVurdering(utfall: Utfall, periode: Periode, kilde: Kilde): Vurdering =
+        Vurdering(
+            utfall = utfall,
+            kilde = kilde,
+            fom = periode.fra,
+            tom = periode.til,
+            vilkår = Vilkår.FRIST_FOR_FRAMSETTING_AV_KRAV,
+            detaljer = "",
+            grunnlagId = null,
+        )
+
+    fun vilkårsvurderFristForFramsettingAvKrav(): List<Vurdering> {
+        // TODO: Må finne ut av hva man skal gjøre når det kommer inn en ny søknad på en behandling som er til beslutter før denne sjekken passerer gjennom tester.
```

Comment:
Kan du lage et trellokort på dette? Eller evt. referere til et. Her må vi ta en vurdering på hva vi ønsker å gjøre.
Endre statusen på behandlingen? Skal vi sperre for å iverksette? Gjøre forskjell på om periodene til tiltakene overlapper/tilstøter eller ikke gjør det?

Og må dette løses i den PRen her, eller vil vi akseptere og merge dette uten denne sjekken. Hva slags logikk må vi endre andre plasser dersom vi legger på denne sjekken?

Reply:
Ja, jeg opprettet et trellokort med beskrivelse av problemet på fredag:
https://trello.com/c/vK4Zc1M9/971-finne-ut-hva-vi-skal-gj%C3%B8re-n%C3%A5r-det-kommer-inn-en-ny-s%C3%B8knad-p%C3%A5-en-behandling-som-er-til-beslutter

Om vi kan merge uten sjekken vet jeg ikke egentlig, det går jo ikke til prod men det må håndteres. Jeg har nok frem til nå tenkt at det er utenfor scope for akkurat denne oppgaven, og at vi uansett må prioritere å gjøre noe med ☝️ som er et problem med eller uten denne featuren.